metricbeat/docker-compose.yml

```diff
@@ -172,3 +215,7 @@ services:
 
   zookeeper:
     build: ./module/zookeeper/_meta
+
+networks:
+  mongonet:
+    driver: bridge
```

Comment:
I'm not sure if out testing framework is ready to support scenarios with multiple nodes in a custom network :slightly_frowning_face: